src/schemas/receipt.yaml

```diff
@@ -119,3 +119,26 @@ ReceiptInfo:
       title: blob gas price
       description: The actual value per gas deducted from the sender's account for blob gas. Only specified for blob transactions as defined by EIP-4844.
       $ref: '#/components/schemas/uint'
+AuthorizationList:
```

Comment:
this value should be in the transaction scheme like AccessList is, not in receipt.yaml

Reply:
Like should I move it there and replace with the current authorizationList right ?